### PR TITLE
Basemodel updates for Geohash and Local cache service fixes

### PR DESCRIPTION
**Pull request by chelm**
A few updates and fixes for supporting geohash aggregation calls to the DB and fixing service registration POSTs when running the local cache.

### CODE REVIEW

**Comment by chelm** [comment]
ping @ngoldman

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "koop",
-  "version": "2.1.4",
+  "version": "2.1.6",
```
Comment (ngoldman) [inline package.json]: @chelm wrong number! should be `2.1.5` right?

Comment (chelm) [inline package.json]: oh hmm

**Comment by chelm** [comment]
@ngoldman too many repo numbers in the brain! thanks

**Comment by ngoldman** [comment]
@chelm one day I will automate that part too

**Comment by ngoldman** [comment]
:+1:

**Pull request merged**